components/scream/src/physics/p3/p3_f90.cpp

```diff
@@ -76,7 +62,7 @@ FortranDataIterator::FortranDataIterator (const FortranData::Ptr& d) {
 
 void FortranDataIterator::init (const FortranData::Ptr& dp) {
   d_ = dp;
-  fields_.reserve(33);
+  fields_.reserve(23);
```

Comment:
If you find this line annoying, you can remove it. It was a silly microoptimization.

Reply:
(On my part, I mean.)

Reply:
Thanks. If I understand correctly, the .reserve(x) just tells the compiler that you will need enough space for x elements in your vector, right? Thanks for the review.

Reply:
It acts at runtime. It makes the capacity of the vector at least that big. That means there won't be reallocs behind the scenes until the capacity is exceeded. But this is just test code, so it's a silly micro-opt.